datafusion/core/tests/sqllogictests/test_files/groupby.slt

```diff
@@ -1960,21 +1960,20 @@ SortPreservingMergeExec: [col0@0 ASC NULLS LAST]
 --SortExec: expr=[col0@0 ASC NULLS LAST]
 ----ProjectionExec: expr=[col0@0 as col0, LAST_VALUE(r.col1) ORDER BY [r.col0 ASC NULLS LAST]@3 as last_col1]
 ------AggregateExec: mode=FinalPartitioned, gby=[col0@0 as col0, col1@1 as col1, col2@2 as col2], aggr=[LAST_VALUE(r.col1)]
---------SortExec: expr=[col0@3 ASC NULLS LAST]
```

Comment:
Generally in the plan `SortExec` before `AggregateExec: mode=FinalPartitioned` or `AggregateExec: mode=Final` is removed.

Reply:
I it makes sense that there is nothing inserted between the two aggregate phases given the aggregates themselves track the sortedness